microcosm_flask/errors.py

```diff
@@ -28,6 +28,17 @@ class ErrorSchema(Schema):
     context = fields.Nested(ErrorContextSchema, required=False)
 
 
+def as_retryable(error):
+    """
+    Given an exception, mark it as retryable when serializing
+    into transporty layer error response.
+
+    """
+    error.retryable = True
```

Comment:
Do you have a test case? I suspect this will work, but I know sometimes you need to use `setattr` if the underlying type isn't friendly to extension. I don't really know when that happens however.